docs/user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -6,15 +6,31 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/introduction-to-pl
+Strapi is built around different types of plugins. Every default Strapi application comes with the following pre-installed plugins:
 
-::: note
-Core plugins which include the Content Manager and the Content-type Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md) and [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the other, non-core plugins.
-:::
+* Content Manager (see [Introduction to the Content Manager](../content-manager/introduction-to-content-manager.md))
+* Content Type Builder (see [Introduction to the Content-type Builder](../content-types-builder/introduction-to-content-types-builder.md))
+* Email
+* Media Library (implemented via the [Upload plugin](../../../developer-docs/latest/plugins/upload.html))
+* Internationalization
+* Roles and Permissions
 
- From the admin panel, administrators are allowed to:
+These plugins are essential for your Strapi application to function and cannot be uninstalled.
 
-- discover additional plugins in the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ (see [Managing Marketplace plugins](/user-docs/latest/plugins/installing-plugins-via-marketplace.md))
-- review the currently installed plugins in ![Plugins icon](../assets/icons/plugins.svg) _Plugins_
+Additional plugins that you can use to extend and customize your Strapi applications are available in the [Marketplace](../plugins/installing-plugins-via-marketplace.md). This section focuses on how to install and manage these additional plugins.
+
+From the admin panel, administrators are allowed to:
+
+- discover additional plugins and [providers](#providers) in the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ (see [Managing Marketplace plugins](/user-docs/latest/plugins/installing-plugins-via-marketplace.md))
+- review the currently installed plugins and [providers](#providers) in ![Plugins icon](../assets/icons/plugins.svg) _Plugins_
 
 ![Plugins settings](../assets/plugins/plugins-settings.png)
+
+## Providers
+
+Some plugins can be further extended through the configuration of _providers_, packages designed to be used on top of an existing plugin and add a specific integration to it. For example, you can use the AWS S3 provider to extend the Media Library plugin and store files in your S3 bucket rather than locally on your server.
+
+Currently, the only plugins designed to work with providers are the:
+
+* [Email plugin](../../../developer-docs/latest/plugins/email.html#configure-the-plugin), and
+* [Media Library plugin](../../../developer-docs/latest/plugins/upload.html#using-a-provider).
```

Comment:
I really liked the "(implemented via the [Upload plugin](../../../developer-docs/latest/plugins/upload.html))" used line 14, maybe we could re-use it here? So people directly jumping to/only reading the Providers section of this document will know.